src/ansys/pre_commit_hooks/add_license_headers.py

```diff
@@ -80,6 +80,12 @@ def set_lint_args(parser):
         help="Default license for headers.",
         default=DEFAULT_LICENSE,
     )
+    parser.add_argument(
+        "--check_license",
+        type=str,
+        help="Whether or not to check for the license in the header.",
+        default="True",
+    )
```

Comment:
I would use it as a flag type. Have a look here: 

https://github.com/ansys/ansys-tools-repo-sync/blob/d2468e036905fe15064d90e2303bffb20b207666/src/ansys/tools/repo_sync/__main__.py#L51-L56

And then, I would use this flag the other way around, something like ``--ignore-license-check``

Reply:
I ended up using the flag option for argparse since it was easier to integrate with the rest of the code